static/src/javascripts/projects/common/modules/commercial/ads/sticky-mpu.js

```diff
@@ -1,43 +1,36 @@
 define([
     'common/utils/config',
     'common/modules/ui/sticky',
-    'common/utils/fastdom-promise',
-    'lodash/objects/defaults'
+    'common/utils/fastdom-promise'
 ], function (
     config,
     Sticky,
-    fastdom,
-    defaults
+    fastdom
 ) {
 
-    var mpuHeight = 275,
-        StickyMpu = function ($adSlot, options) {
-            this.$adSlot = $adSlot;
-            this.opts    = defaults(options || {}, {
-                top: 0
-            });
-        };
+    var MPU_HEIGHT = 275;
```

Comment:
Just a convention from back when I used to write C :blush: 

I recently switched back to it because we're not far from using `const` and `let` instead of `var` and this way it's going to be a little easier to discriminate between the two.